.github/workflows/release.yml

```diff
@@ -0,0 +1,88 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  trigger-buildkite-pipeline:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger a Buildkite Build
+        uses: "buildkite/trigger-pipeline-action@v2.0.0"
+        with:
+          buildkite_api_access_token: ${{ secrets.TRIGGER_BK_BUILD_TOKEN }}
+          pipeline: "anza/agave-secondary"
+          branch: "${{ github.ref_name }}"
+          commit: "HEAD"
+          message: ":github: Triggered from a GitHub Action"
+
+  draft-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Release
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.repos.createRelease({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              tag_name: '${{ github.ref_name }}',
+              name: 'Release ${{ github.ref_name }}',
+              body: '🚧',
+              draft: true,
+              prerelease: false
+            })
+
+  version-bump:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Parse Info
+        id: parse_info
+        run: |
+          # get the next version
+          version=${{ github.ref_name }}
```

Comment:
I don't know if it will work here, but scripts/confirm-cargo-version-numbers-before-bump.sh was written to confirm some assumptions that should be true here.